models/userDao.js

```diff
@@ -0,0 +1,42 @@
+const appDataSource = require("./appDataSource");
+
+const createUser = async (email, name, password, address, phone, birthdate) => {
+  try {
+    return await myDataSource.query(
```

Comment:
```suggestion
    return appDataSource.query(
```
1. 위에서 모듈로 가져오는 dataSource를 변수명 appDataSource로 작성하였기 때문에 myDataSource.query()라는 코드는 작동하지 않습니다.
2. 동기적으로 실행되는걸 기다린 뒤 추가적으로 수행할 코드(로직)가 없기 떄문에 await는 작성되지 않아도 됩니다.

Reply:
네 확인하고 수정했습니다.